### PR TITLE
br: support session-token and assume-role for backup & restore

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: external-storage-uri.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ summary: 介绍了外部存储服务 Amazon S3、GCS、和 Azure Blob Storage 
 
     - `access-key`：访问密钥
     - `secret-access-key`：秘密访问密钥
-    - `session-token`：临时会话令牌（BR 尚不支持该参数）
+    - `session-token`：临时会话令牌
```
Comment (BornChanger) [inline external-storage-uri.md:22]:
```suggestion
    - `session-token`：临时会话令牌 (BR v7.6 版本及以上支持）
```

Comment (BornChanger) [reply external-storage-uri.md:22]: 第 30 行, `role-arn` 这里也要加上类似的表述

**Review by BornChanger — APPROVED** [review]

File: releases/release-7.6.0.md [hunk 2]
```diff
@@ -356,7 +356,7 @@ TiDB 版本：7.6.0
 + Tools
 
     + Backup & Restore (BR)
-
+	- 新增了对全量备份恢复阶段对 s3 session-token 以及 assume-role 的认证支持。[#39832](https://github.com/pingcap/tidb/issues/39832) @[3pointer](https://github.com/3pointer)
```
Comment (qiancai) [inline releases/release-7.6.0.md:359]:
```suggestion
	- 新增了全量备份恢复阶段对 Amazon S3 `session-token` 以及 `assume-role` 的认证支持 [#39832](https://github.com/pingcap/tidb/issues/39832) @[3pointer](https://github.com/3pointer)
```

**Comment by BornChanger** [comment]
@3pointer please address the left comments.

File: external-storage-uri.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ summary: 介绍了外部存储服务 Amazon S3、GCS、和 Azure Blob Storage 
 
     - `access-key`：访问密钥
     - `secret-access-key`：秘密访问密钥
-    - `session-token`：临时会话令牌（BR 尚不支持该参数）
+    - `session-token`：临时会话令牌（BR v7.6.0 版本及以上支持）
```
Comment (qiancai) [inline external-storage-uri.md:22]:
```suggestion
    - `session-token`：临时会话令牌（BR v7.6.0 及之后版本支持）
```

File: external-storage-uri.md [hunk 4]
```diff
@@ -19,15 +19,15 @@ summary: 介绍了外部存储服务 Amazon S3、GCS、和 Azure Blob Storage 
...
     - `storage-class`：上传对象的存储类别，例如 `STANDARD`、`STANDARD_IA`
     - `sse`：加密上传的服务端加密算法，可以设置为空、`AES256` 或 `aws:kms`
     - `sse-kms-key-id`：如果 `sse` 设置为 `aws:kms`，则使用该参数指定 KMS ID
     - `acl`：上传对象的标准 ACL (Canned ACL)，例如 `private`、`authenticated-read`
-    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)。
+    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)（BR v7.6.0 版本及以上支持）。
```
Comment (qiancai) [inline external-storage-uri.md:30]:
```suggestion
    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)（BR v7.6.0 及之后版本支持）。
```

File: external-storage-uri.md [hunk 5]
```diff
@@ -19,15 +19,15 @@ summary: 介绍了外部存储服务 Amazon S3、GCS、和 Azure Blob Storage 
...
     - `storage-class`：上传对象的存储类别，例如 `STANDARD`、`STANDARD_IA`
     - `sse`：加密上传的服务端加密算法，可以设置为空、`AES256` 或 `aws:kms`
     - `sse-kms-key-id`：如果 `sse` 设置为 `aws:kms`，则使用该参数指定 KMS ID
     - `acl`：上传对象的标准 ACL (Canned ACL)，例如 `private`、`authenticated-read`
-    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)。
+    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)（BR v7.6.0 及之后版本支持）。
```
Comment (qiancai) [inline external-storage-uri.md:30]: @3pointer 要不要单独再提个 v7.5 分支的 PR，在 v7.5 分支里说明 BR 尚不支持该参数

Comment (qiancai) [inline external-storage-uri.md:30]:
```suggestion
    - `role-arn`：当需要使用特定的 [IAM 角色](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles.html)来访问第三方 Amazon S3 的数据时，使用这个参数来指定 IAM 角色的对应 [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/zh_cn/general/latest/gr/aws-arns-and-namespaces.html)（例如 `arn:aws:iam::888888888888:role/my-role`）。关于使用 IAM 角色访问第三方 Amazon S3 数据的场景，请参考 [AWS 相关文档介绍](https://docs.aws.amazon.com/zh_cn/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)。（BR v7.6.0 及之后版本支持）
```

**Review by qiancai — APPROVED** [review]

File: releases/release-7.6.0.md [hunk 6]
```diff
@@ -357,6 +357,7 @@ TiDB 版本：7.6.0
 
     + Backup & Restore (BR)
 
+        - 新增了全量备份恢复阶段对 Amazon S3 `session-token` 以及 `assume-role` 的认证支持 [#39832](https://github.com/pingcap/tidb/issues/39832) @[3pointer](https://github.com/3pointer)
```
Comment (qiancai) [inline releases/release-7.6.0.md:360]:
```suggestion
        - 新增全量备份恢复阶段对 Amazon S3 `session-token` 以及 `assume-role` 的认证支持 [#39832](https://github.com/pingcap/tidb/issues/39832) @[3pointer](https://github.com/3pointer)
```

**Pull request merged**